build/tools/amebasmart/gnu_utility/km0_km4_app_RELEASE_NOTE.txt

```diff
@@ -1,5 +1,11 @@
 /* == "version" + "Realtek git version" + "compile date" + "compile time" == */
 
+== version c7e8f3b164 2024/11/22-07:56:05 ==
```

Comment:
Could you check PR #6525 ? That also has a change on km0_km4_app. That is missing here.

Reply:
A rebase is needed, Hnery is working on that.

Reply:
@lhenry-realtek @jwei5 changed them already. Need to update them again.